inputfiles/addedTypes.jsonc

```diff
@@ -11,6 +11,19 @@
                     ]
                 }
             },
+            "Comment": {
```

Comment:
There's no mixin named Comment and thus this doesn't do anything. Comment inherits from CharacterData so this is not needed either.